frontend/webpack.config.ts

```diff
@@ -78,13 +79,14 @@ const config: Configuration = {
         exclude: /node_modules\/(?!(bitbucket|ky)\/)/,
         use: [
           { loader: 'cache-loader' },
-          {
-            loader: 'thread-loader',
-            options: {
-              // Leave one core spare for fork-ts-checker-webpack-plugin
-              workers: require('os').cpus().length - 1,
```

Comment:
This seems to be the default number of workers for thread-loader without needing to be explicitly set.

https://webpack.js.org/loaders/thread-loader/